dawn/test/integration-test/TestIIRDeserializer.cpp

```diff
@@ -265,4 +267,4 @@ TEST(IIRDeserializerTest, LaplStencil) {
   compareIIRs(lap_stencil_from_file, lap_stencil_memory);
 }
 
-} // anonymous namespace
\ No newline at end of file
+} // anonymous namespace
```

Comment:
Make sure that you properly use clang-format (and it should not be somehow interact with the IDE adding the new line). I thought it should remove the new line at the end of the file.

Reply:
Probably the IDE's clang-format configuration is messed up ... but these kind of problems should be solved with the pre-commit hook right?